src/types/widget-data/line.ts

```diff
@@ -45,4 +46,6 @@ export type LineSeries<T = any> = BaseSeries & {
     };
     /** Options for the point markers of line series */
     marker?: LineMarkerOptions;
+    dashStyle?: DashStyle;
+    linecap?: LineCap;
```

Comment:
Perhaps it is worth saying a few words in the comments about the settings and default values?